src/_posts/databases/postgresql/2000-01-01-monitoring.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Monitoring and Auditing Your Scalingo for PostgreSQL® Addon
+nav: Monitoring and Auditing
+modified_at: 2023-11-28 00:00:00
+tags: databases postgresql addon
+index: 4
+---
+
+
+Each Scalingo for PostgreSQL® addon comes with several tools allowing for basic
```

Comment:
suggestion: I find the use of the word "basic" slightly reductive. Plus it does not add anything. I would remove it.

```suggestion
Each Scalingo for PostgreSQL® addon comes with several tools allowing for
```